app/views/publish/courses/a_level_requirements/what_a_level_is_required/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :page_title, title_with_error_prefix("SOME PAGE TITLE", @errors && @errors.any?) %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to("change-me") %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @wizard.current_step, url:
+        publish_provider_recruitment_cycle_course_a_levels_are_any_a_levels_required_for_this_course_path(
+          @provider.provider_code,
+          @provider.recruitment_cycle_year,
+          @course.course_code
+        ) do |form| %>
+
+      <%= form.submit t("continue"), class: "govuk-button" %>
+    <% end %>
+
+    <p class="govuk-body">
+      <%= govuk_link_to(t("cancel"), publish_provider_recruitment_cycle_course_path(@provider.provider_code, @provider.recruitment_cycle_year, @course.course_code)) %>
+    </p>
+  </div>
+</div>
```

Comment:
This whole page is temporary and will be done under the next PR and next Trello card. https://trello.com/c/0fmMekAP/1769-a-levels-2-7-add-a-level-subject-and-grade-question-page-for-tda-courses-on-publish?filter=label:Epic%3A%20TDA